package.json

```diff
@@ -25,114 +25,112 @@
     "prepare": "./scripts/link-them.sh"
   },
   "dependencies": {
-    "@glimmer/syntax": "^0.42.0",
+    "@glimmer/syntax": "^0.42.2",
     "broccoli-bridge": "^1.0.0",
     "broccoli-caching-writer": "^3.0.3",
     "broccoli-debug": "^0.6.4",
     "broccoli-filter": "^1.2.4",
     "broccoli-funnel": "^2.0.2",
     "broccoli-merge-trees": "^3.0.1",
     "broccoli-persistent-filter": "^2.3.1",
-    "broccoli-plugin": "^1.3.1",
-    "broccoli-source": "^1.1.0",
-    "broccoli-stew": "^2.0.0",
+    "broccoli-plugin": "^3.0.0",
+    "broccoli-source": "^3.0.0",
+    "broccoli-stew": "^3.0.0",
     "chalk": "^2.4.2",
     "ember-angle-bracket-invocation-polyfill": "^2.0.2",
     "ember-assign-polyfill": "^2.6.0",
-    "ember-auto-import": "^1.5.2",
+    "ember-auto-import": "^1.5.3",
     "ember-cli-autoprefixer": "^0.8.1",
-    "ember-cli-babel": "^7.7.3",
-    "ember-cli-clipboard": "^0.11.1",
-    "ember-cli-htmlbars": "^3.0.1",
-    "ember-cli-htmlbars-inline-precompile": "^2.1.0",
+    "ember-cli-babel": "^7.12.0",
+    "ember-cli-clipboard": "^0.13.0",
+    "ember-cli-htmlbars": "^4.0.7",
     "ember-cli-postcss": "^5.0.0",
-    "ember-cli-sass": "10.0.0",
-    "ember-cli-string-helpers": "^1.9.0",
+    "ember-cli-sass": "10.0.1",
+    "ember-cli-string-helpers": "^4.0.5",
     "ember-cli-string-utils": "^1.1.0",
     "ember-code-snippet": "^2.4.1",
     "ember-component-css": "^0.7.4",
     "ember-composable-helpers": "^2.3.1",
     "ember-concurrency": "^0.9.0 || ^0.10.0 || ^1.0.0",
-    "ember-data": "2.x - 3.x",
-    "ember-fetch": "^6.7.0",
+    "ember-data": "^3.13.1",
```

Comment:
Do we want to enforce 3.13.1+ for ember-data? Since these are dependencies, and not devDependencies, I think keeping the range might be nice, so theoretically, it'll use whatever version you may already have. Not really sure if it makes a huge difference though. @samselikoff thoughts?

Reply:
That's fair. I will update.

Reply:
I think we should also probably keep the deps at whatever the blueprints set them to for addons, so we do not enforce a higher version unnecessarily, but I am not sure if it's worth worrying about. Just trying to keep things compatible with a wide range of versions out of the box.

Reply:
Yep more liberal ranges are probably good, though I think https://github.com/ember-learn/ember-cli-addon-docs/pull/409 is the higher yield work to address this sorta thing.